runtime/altair/src/lib.rs

```diff
@@ -223,8 +223,8 @@ impl Contains<Call> for BaseCallFilter {
 }
 
 parameter_types! {
-	pub const ReservedXcmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT / 4;
-	pub const ReservedDmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT / 4;
+	pub const ReservedXcmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT.saturating_div(4);
```

Comment:
Is these changes done automatically by the upgrade tool? This division is checked at compile time so I think there is no need to check if saturates.

Reply:
I don't know, just picked this up - cc @gruberb?

Reply:
I wasn't sure about this one yet @lemunozm. The normal `div` is not `const`, so I needed another way of dividing and assigning it to a `const`. 

I am not sure, I remember @mustermeiszer opened a PR to add `const div` against the substrate repo?

Reply:
I think `div` is const if both arguments are const. I've done a fast example in [playrust](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=8eb84c3b4cb2a4a4f3ab1a97b9c05af1)

Reply:
We are getting this error here:

```
error[E0015]: cannot call non-const operator in constants
   --> libs/primitives/src/lib.rs:204:43
    |
204 |     pub const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND / 2;
    |                                              ^^^^^^^^^^^^^^^^^^^^^
    |
note: impl defined here, but it is not `const`
   --> /Users/gruberbastian/.cargo/git/checkouts/substrate-7e08433d4c370a21/cc370aa/frame/support/src/weights/weight_v2.rs:236:1
    |
236 | / impl<T> Div<T> for Weight
237 | | where
238 | |     u64: Div<T, Output = u64>,
239 | |     T: Copy,
...   |
244 | |     }
245 | | }
    | |_^
    = note: calls in constants are limited to constant functions, tuple structs and tuple variants
```

Reply:
mmm... Ok, I see that `Weight` it's no longer an `u64`. And for custom types `div` is not const ([yet](https://doc.rust-lang.org/beta/unstable-book/library-features/const-ops.html)). 
Sorry for the noise and thanks for testing 🤗

Reply:
Thanks for chipping in @lemunozm 🤘🏻

Reply:
@lemunozm @NunoAlexandre there is a PR that it already merged where we have a const div possibility. https://github.com/paritytech/substrate/pull/12322

But this will only hit v0.9.30

Reply:
Love it!